AutoCollection/WebSnippet/fetchSnippet.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
```

Comment:
Remove this file as snippet is already part of the changes and will be checked in until npm package is available